packages/discord.js/src/util/Permissions.js

```diff
@@ -17,160 +18,42 @@ class Permissions extends BitField {
 
   /**
    * Data that can be resolved to give a permission number. This can be:
-   * * A string (see {@link Permissions.FLAGS})
+   * * A string (see {@link PermissionFlagsBits})
    * * A permission number
    * * An instance of Permissions
    * * An Array of PermissionResolvable
    * @typedef {string|bigint|Permissions|PermissionResolvable[]} PermissionResolvable
    */
 
-  /**
-   * Gets all given bits that are missing from the bitfield.
-   * @param {BitFieldResolvable} bits Bit(s) to check for
-   * @param {boolean} [checkAdmin=true] Whether to allow the administrator permission to override
-   * @returns {string[]}
-   */
-  missing(bits, checkAdmin = true) {
-    return checkAdmin && this.has(this.constructor.FLAGS.ADMINISTRATOR) ? [] : super.missing(bits);
-  }
```

Comment:
This can probably be brought back either as it was before or returning a bitfield, right?